tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -902,6 +902,12 @@ public TezConfiguration(boolean loadDefaults) {
           TEZ_AM_PREFIX + "dag.appcontext.thread-count-limit";
 
   public static final int TEZ_AM_DAG_APPCONTEXT_THREAD_COUNT_LIMIT_DEFAULT = 10;
+  /** Boolean value. Instructs AM to delete intermediate attempt data for failed task attempts */
```

Comment:
1. nit: leave 1 line space between conf properties
2. put this right after TEZ_AM_DAG_CLEANUP_THREAD_COUNT_LIMIT in order to get cleanup related properties close to each other
3. there is a comment on TEZ_AM_DAG_CLEANUP_THREAD_COUNT_LIMIT:
```
   * Int value. Upper limit on the number of threads used to delete DAG directories on nodes.
```
as far as I can see this patch doesn't introduce a new executor for cleaning up failed attempt data, instead, it uses DeletionTrackerImpl.dagCleanupService, so this comment should be changed to reflect that this thread count is shared between dag cleanup + failed attempt cleanup

Reply:
ack